README.md

```diff
@@ -24,15 +24,16 @@ Let this module handle file creation and systemd reloading.
 Or handle file creation yourself and trigger systemd.
 
 ```puppet
-include ::systemd
+include ::systemd::systemctl::daemon_reload
```

Comment:
We can contain `daemon_reload` but we can't contain `tmpfiles` since tmpfiles is triggered from the use of the `tmpfile` defined type.

A notify to `Class[systemd]` makes sense for reloading the whole thing though so I'll switch that one to a contain and hope that we don't get more creative in the future.